### PR TITLE
[.NET] Turkish DateTime Set support

### PR DESCRIPTION
**Pull request by LionbridgeCSII**
All test cases pass.

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Text.DateTime/Turkish/Parsers/TurkishSetParserConfiguration.cs [hunk 1]
```diff
@@ -80,23 +80,23 @@ public bool GetMatchedDailyTimex(string text, out string timex)
         {
             var trimmedText = text.Trim();
 
-            if (trimmedText.Equals("daily"))
+            if (trimmedText.Equals("günlük"))
```
Comment (tellarin) [inline .NET/Microsoft.Recognizers.Text.DateTime/Turkish/Parsers/TurkishSetParserConfiguration.cs]: Please move such hardcoded values to YAML.

File: Patterns/Turkish/Turkish-DateTime.yaml [hunk 2]
```diff
@@ -123,6 +123,7 @@ DateUnitRegex: !simpleRegex
 DateTokenPrefix: ''
 TimeTokenPrefix: ''
 TokenBeforeDate: ''
+TokenBeforeDate1: '''de'
```
Comment (tellarin) [inline Patterns/Turkish/Turkish-DateTime.yaml]: Why declare a new entry here?

File: .NET/Microsoft.Recognizers.Text.DateTime/Turkish/Parsers/TurkishSetParserConfiguration.cs (hunk 1 above)
Comment (LionbridgeCSII) [reply .NET/Microsoft.Recognizers.Text.DateTime/Turkish/Parsers/TurkishSetParserConfiguration.cs]: done (added named groups to PeriodicRegex and DateUnitRegex and used those regexes to extract the values).

File: Patterns/Turkish/Turkish-DateTime.yaml (hunk 2 above)
Comment (LionbridgeCSII) [reply Patterns/Turkish/Turkish-DateTime.yaml]: To reduce merge conflicts, the yaml file already includes the changes from the DateTimePeriod PR. 
This new entry (now changed to TokenListBeforeDate) is required only by DateTimePeriodParser, while all other subtypes need to use the standard empty TokenBeforeDate.

**Pull request merged**